### PR TITLE
Start breaking out repeated content into xincludes

### PR DESCRIPTION
**Pull request by cjschroder**
which allows embedding repeated content, rather than creating 
cross-references. And reduce the numbers of repeated warnings, which
are annoying and cluttery. This is the first of multiple PRs, to keep them at manageable sizes.

### CODE REVIEW

**Comment by cjschroder** [comment]
Re https://github.com/SUSE/doc-cap/issues/319 and https://github.com/SUSE/doc-cap/issues/317

**Comment by cjschroder** [comment]
Here is a PDF--
[book.cap.guides_color_en.pdf](https://github.com/SUSE/doc-cap/files/2833888/book.cap.guides_color_en.pdf)

File: xml/cap_depl_install_caasp.xml [hunk 1]
```diff
@@ -320,6 +289,11 @@ Happy Helming!</screen>
      </para>
     </listitem>
    </itemizedlist>
+   <para>
+       &productname; includes the necessary PSP configurations in the &helm; 
+       charts to run on &suse; &caasp;,
+       and are set up automatically without requiring manual configuration.
```
Comment (btat) [inline xml/cap_depl_install_caasp.xml]: Need a TODO/placeholder to insert info on custom PSPs

**Comment by cjschroder** [comment]
Merging; we do not have time to fiddle around!

**Pull request merged**